Zend/zend_virtual_cwd.c

```diff
@@ -603,6 +603,7 @@ static size_t tsrm_realpath_r(char *path, size_t start, size_t len, int *ll, tim
 			if (!pathw) {
 				return (size_t)-1;
 			}
+			PHP_WIN32_IOUTIL_CHECK_PATH_W(pathw, (size_t)-1, 1);
```

Comment:
So I was confused by the ``(size_t)-1`` as I didn't know one could do that.
But seems ``SIZE_MAX`` exists and is a standard C99 definition, so maybe use that instead?

Reply:
Actually, I'm just tired, and now that I see the macro expansion again it makes perfect sense, so disregard this .-.

Reply:
We use `(size_t)-1` quite a lot throughout php-src (104 occurrences), and there isn't a real problem doing that; we sometimes use `(size_t)~0` instead. But right, why not use `SIZE_MAX`? That should probably be done in a separate PR, though.